Rs-Project9/ZooLists.java

```diff
@@ -0,0 +1,78 @@
+package ZooManagementSystem;
+
+import java.util.*;
+
+enum AnimalEnum {
+    OneAnimal("Lion", "Mammal"), TwoAnimal("Tiger", "Mammal"), ThreeAnimal("Pigeon", "Bird"), 

```

Comment:
change the enum var names..oneanimal twoanimal does not describe anything about it.